sympy/ntheory/factor_.py

```diff
@@ -1545,3 +1545,40 @@ def core(n, t=2):
         for p, e in factorint(n).items():
             y *= p**(e % t)
         return y
+
+
+def digits(n, b=10):
+    """
+    Return a list of the digits of n in base b. First item in list is base hint.
```

Comment:
First item ... -> The first element in the list is b (or -b if n is negative).

I think you can delete the whole 2nd paragraph as it contains redundancies and information that is available in the Examples.